### DISCUSSION TITLE
How to use CDN method Editorjs

### DISCUSSION THREAD

**Discussion by atomic-coderx** [codex-team/editor.js#2441] [Q&A]
Hello guys am having issue loading Editorjs with import since I get my server  responding to it as a html/text file which the browser was not able to handle. Well it's an urgent project so I need to fix it first (am using lampp for development) using CDN method. Please any idea how I can use CDN to implement Editorjs? Thanks

**Comment by neSpecc** [comment]
Configure your Apache (Nginx, or other web-server you use) to serve `.js` files with `Content-Type: application/javascript` response header

**Reply by atomic-coderx** [reply]
Alright thanks
